source/DatePicker.js

```diff
@@ -262,7 +253,7 @@
 					second: valueSeconds,
 					millisecond: valueMilliseconds
 				});
-				this.setValue(this.localeValue.getJSDate());
+				this.setValue(localeValue);
```

Comment:
Is it still necessary to do this, or can we just update the existing `ilib.Date` object?